proto/xudrpc.proto

```diff
@@ -592,6 +592,8 @@ message PlaceOrderRequest {
   string order_id = 4 [json_name = "order_id"];
   // Whether the order is a buy or sell.
   OrderSide side = 5 [json_name = "side"];
+  // The local id of a previous order to be replaced (optional).
```

Comment:
I think the "optional" here is implied. And maybe say "existing" order instead of previous?

On second thought, I think even better would be to clarify here that *if* `replace_order_id` is provided, the order must be successfully found and removed before the new order is placed otherwise an error is returned.